src/components/dependencies/RescoringModal.js

```diff
@@ -1864,20 +2008,22 @@ Rescoring.propTypes = {
   rescoringFeature: PropTypes.object,
   sprints: PropTypes.arrayOf(PropTypes.object).isRequired,
   scanConfig: PropTypes.object,
+  setRescoringsLoading: PropTypes.func.isRequired,
+  rescoringsLoading: PropTypes.bool.isRequired,
+  severityCfgs: PropTypes.arrayOf(PropTypes.object).isRequired,
 }
 
 
 const Rescore = ({
   rescorings,
-  type,
   handleClose,
   setShowProgress,
   scope,
   fetchComplianceData,
   fetchComplianceSummary,
 }) => {
   const [isLoading, setIsLoading] = React.useState(false)
-  const title = findTypedefByName({name: type}).friendlyName
+  const title = 'Rescoring'
```

Comment:
Just remove the `title` variable here and its subsequent usages. This still stems from a type specific rescoring modal whereas it now displays `Apply Rescoring Rescoring` for example.

Reply:
agree